lit-mar/lambda_function.py

```diff
@@ -0,0 +1,124 @@
+from __future__ import print_function
+
+import urllib
+import sys
+import os
+import json
+import subprocess
+
+class Context(object):
+    def __init__(self, logger, metrics_writter, request, response, system_info):
+        self.logger = logger
+        self.metrics_writter = metrics_writter
+        self.request = request
+        self.response = response
+        self.system_info = system_info
+
+
+class BaseResponse(object):
+    """BaseResponse if the object providing standard response attributes and operations"""
+    def __init__(self, response_body=""):
+        """Constructor, construct a response
+
+        Parameters
+        ----------
+        response_body: Optionally give the response a non-empty body
+        """
+        self.headers = []
+        self.status_code = 200
+        self.body = response_body
+
+    def get_dict(self):
+        """Get the response in dict form"""
+        return {"headers":dict(self.headers), "body":self.body, "statusCode":self.status_code}
+
+    def add_response_property(self, key, value):
```

Comment:
Is this used anywhere? AFAIK lambda proxy response doesn't have any new properties. Developer can add the parameters in the response itself and take it from there. Consider removing this helper function as its not used anywhere else in the code - 
See 
https://refactoring.guru/smells/dead-code
https://softwareengineering.stackexchange.com/questions/103031/should-i-remove-unreferenced-code

Reply:
This is to meet MAR spec. MAR may call it (which we need to provide interface but it's up to them to decide whether or not to call). See here https://corp.quip-amazon.com/6PrOAOQTBBHE

Reply:
As this is not used anywhere - I would still recommend removing it. For compliance with MAR we can add these additional helper methods as needed in the future

Reply:
It's not not used anywhere. MAR may call it (we don't have control over MAR code). MAR is a module downloaded and imported.